ui/listener-api/src/main/scala/pl/touk/nussknacker/ui/listener/DeploymentComment.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.ui.listener
+
+import cats.data.Validated
+import cats.data.Validated.{Invalid, Valid}
+
+import scala.language.implicitConversions
+
+trait Comment {
+  def value: String
+}
+
+case class DeploymentComment private(value: String) extends Comment {
+
+  def deployedDeploymentComment: DeploymentComment = withPrefix(DeploymentComment.PrefixDeployedDeploymentComment)
+
+  def canceledDeploymentComment: DeploymentComment = withPrefix(DeploymentComment.PrefixCanceledDeploymentComment)
+
+  private def withPrefix(prefix: String): DeploymentComment = copy(prefix + value)
+}
+
+object DeploymentComment {
+
+  val PrefixDeployedDeploymentComment = "Deployment: "
+  val PrefixCanceledDeploymentComment = "Stop: "
+
+  val FinishedDeploymentComment = new DeploymentComment("Scenario finished")
+
+  def apply(comment: String, settings: Option[DeploySettings]): Validated[CommentValidationError, DeploymentComment] = {
```

Comment:
It is used only in `validateDeploymentComment` which is never used? Or my IDE is cheating me? :)

Reply:
accidentally renamed method and didn't rename usage yesterday, didn't notice, my bad, should be ok now